app/services/participants/transfer/npq.rb

```diff
@@ -0,0 +1,49 @@
+# frozen_string_literal: true
+
+module Participants
+  module Transfer
+    class NPQ
+      def initialize(external_identifier:, new_npq_lead_provider_id:)
+        @external_identifier = external_identifier
+        @new_npq_lead_provider_id = new_npq_lead_provider_id
+      end
+
+      def call
+        return unless npq_lead_provider && participant_identity
+
+        transfer_participant
+      end
+
+    private
+
+      attr_reader :external_identifier, :new_npq_lead_provider_id
+
+      def transfer_participant
+        return unless participant_profile && npq_application
+
+        npq_application.update!(npq_lead_provider:)
+        Rails.logger.info "Participant (#{external_identifier}) was transferred to (#{npq_lead_provider.name}) successfully"
+      end
+
+      def npq_lead_provider
+        @npq_lead_provider ||= NPQLeadProvider.find_by(id: new_npq_lead_provider_id)
```

Comment:
I'd use a find_by! or add some validation otherwise we'd never never now the participant has not been transferred, unless you stare at the logs and look for that one line in the logs, which in prod might a challenging task.
```suggestion
        @npq_lead_provider ||= NPQLeadProvider.find_by!(id: new_npq_lead_provider_id)
```